File: Scenario_1/Automation/Java_Tests_case2_case6/src/test/java/Pages/HomePage.java
package Pages;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.support.ui.ExpectedConditions;

//BasePage'den driver çekildi
public class HomePage extends BasePage {
    public HomePage(WebDriver driver) {
        super(driver);
    }

    //Test burada başlıyor
    public void mizu () {
        driver.get("https://www.mizu.com/en-mx/"); //anasayfaya gidildi

        //pop-up'lar görününce kapatıldı
        wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("// *[@class ='icon-close']"))).click();
        wait.until(ExpectedConditions.visibilityOfElementLocated
                (By.xpath("// *[@class ='policy-popup__close icon-close js-policy-close']"))).click();
        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("[data-module-id='2']")));
    }
}

File: Scenario_1/Automation/dotNetCore_Specflow_Tests_case1_case4/Pages/HomePage.cs
using System;
using OpenQA.Selenium;
using OpenQA.Selenium.Support.UI;
using TechTalk.SpecFlow;

namespace mizu_test {
    [Binding]
public class HomePage  :   BeforeandAfter {     
    //BeforeandAfter sayfasından setup ve teardownları driver ile birlikte çekildi
        public void homepage()
        {
            driver.Navigate().GoToUrl("https://www.mizu.com/");
            WebDriverWait wait = new WebDriverWait(driver, TimeSpan.FromSeconds(8));
            wait.Until(driver => driver.FindElement(By.XPath("// *[@class ='icon-close']")));
            driver.FindElement(By.XPath("// *[@class ='icon-close']")).Click();
            driver.FindElement(By.XPath("// *[@class ='policy-popup__close icon-close js-policy-close']")).Click();
            //siteye gidildi. Pencere büyütüldü ve pop-uplar kapatıldı
        }
        
    }
}